mobile/package.json

```diff
@@ -30,65 +30,64 @@
     "appcenter-analytics": "^1.10.0",
     "appcenter-crashes": "^1.10.0",
     "appcenter-push": "^1.10.0",
-    "babel-plugin-transform-inline-environment-variables": "^0.3.0",
-    "immutable": "^3.8.2",
-    "moment": "^2.20.1",
-    "react": "^16.5.0",
-    "react-native": "^0.57.2",
+    "babel-plugin-transform-inline-environment-variables": "^0.4.3",
+    "immutable": "^4.0.0-rc.12",
```

Comment:
how did you manage to update to release candidate? :D

Reply:
afraid of javascript libraries, of everything by facebook, and of release candidates. That's all three :) If it works with 3.x i'd stick to 3.x

Reply:
But it already works with 'rc' and at the same time I'm not sure it would work with 3.x.